src/index.html

```diff
@@ -4,10 +4,472 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Dia</title>
+    <link
+    rel="icon"
+    type="image/x-icon"
+    href="./images/logo/favicon.svg">
```

Comment:
Let's fix code style everywhere
```suggestion
    <link
      rel="icon"
      type="image/x-icon"
      href="./images/logo/favicon.svg"
    >
```